gh-repo-export

```diff
@@ -176,6 +185,22 @@ start_migration() {
   MIGRATIONS[$MIGRATION_ID]="pending"
 }
 
+if $SKIP_DOWNLOAD; then
+  if ! type -p curl >/dev/null; then
+    die "'curl' is required for --skip-download option and could not be found"
+  fi
+
+  if ! type -p sed >/dev/null; then
+    die "'sed' is required for --skip-download option and could not be found"
+  fi
+
+  if [[ $ARCHIVE != "" ]]; then
+    die "--archive cannot be used with --skip-download"
+  fi
+
+  TOKEN=$(gh auth token)
```

Comment:
@timrogers : what do you think about calling this as a subexpression down below closer to grabbing the URL?

Reply:
fi or tokens 

> @timrogers: نظر شما در مورد فراخوانی این عبارت به عنوان یک عبارت فرعی در زیر نزدیکتر به گرفتن URL چیست؟

Reply:
👍🏻